plugins/catalog-import/src/components/StepPrepareCreatePullRequest/PreparePullRequestForm.tsx

```diff
@@ -32,7 +32,7 @@ type Props<TFieldValues extends Record<string, any>> = Pick<
   render: (
     props: Pick<
       UseFormMethods<TFieldValues>,
-      'errors' | 'register' | 'control'
+      'errors' | 'register' | 'control' | 'formState' | 'watch'
```

Comment:
The values prop already provides the value of watch(). Also why do we need formState?

Reply:
Forgot to remove. My first time using `react-hook-form` so I was attempting to use it to toggle the button disabled property using `isValid` rather then the multiple checks it's using now. I'm used to Angular forms where there are more concepts like `pristine` and an aggregate `touched` which didn't work out. I essentially wanted the `submit` button to be enabled until the form was dirtied or invalid. Couldn't get the right mix to work. I'll remove this, unless you have an alternative suggestion.

Reply:
That's a tricky one. The dirty check is implemented this way for the input-url-form but I couldn't get it to work in this create-or-form. I struggled with the fact that it is using default values for some fields and that non-touched defaults are interpreted as them being not dirty. So this would break if the defaults are already valid and I skipped this behavior.